spec/index.html

```diff
@@ -617,6 +633,132 @@ <h2>Streams</h2>
+        YAMl <a>block sequences</a> and <a>flow sequences</a>.
+        Both forms describe the same <a>array</a> of two <a>strings</a>.</p>
+
+      <p>A YAML <a>block sequence</a>:</p>
+      <pre data-transform="updateExample"
+           data-content-type="application/yaml">
+        <!--
+          - one
+          - two
+        -->
+      </pre>
+
+      <p>A YAML <a>flow sequence</a>:</p>
+      <pre data-transform="updateExample"
+           data-content-type="application/yaml">
+        <!--
+          [one, two]
+        -->
+      </pre>
+    </aside>
+
+    <p>The conversion process represented here is compatible with
+      the description of <a data-cite="YAML#constructing-native-data-structures">Constructing Native Data Structures</a> in [[YAML]].</p>
+
+    <section id="convert-stream" class="algorithm">
+      <h3>Converting a <a>YAML stream</a></h3>
+
+      <p>A <a>YAML stream</a> is composed of zero or more <a>YAML documents</a>.</p>
+
+      <p class="issue" data-number="63">
+        <a>YAML streams</a> may correspond more directly to
+        [[[RFC7464]]], which are not presently part of the
+        JSON-LD representation model.
+        The description here more closely aligns with how JSON-LD
+        interprets <a data-cite="JSON-LD11-API#html-content-algorithms">HTML Scripts</a>.
+      </p>
+      <ol>
+        <li>Set <var>stream content</var> to an empty <a>array</a>.</li>
+        <li>If the <a>stream</a> is empty,
+          set <var>stream content</var> to an empty <a>array</a>.</li>
+        <li>Otherwise, if the <a>stream</a> contains a single <a>YAML document</a>,
+          set <var>stream content</var> the result of
+          <a href="#convert-document" class="sectionRef"></a>.</li>
+        <li>Otherwise: for each <var>document</var> in the <a>stream</a>:
+          <ol>
+            <li>Set <var>doc</var> to the result of <a href="#convert-document" class="sectionRef"></a> for <var>document</var>.</li>
+            <li>If <var>doc</var> is an <a>array</a>,
+              merge it to the end of <var>stream content</var>.</li>
+            <li>Otherwise, append <var>doc</var> to <var>stream content</var></li>
+          </ol>
+        </li>
+        <li>The conversion result is <var>stream content</var>.</li>
+      </ol>
+      <p>Any error reported in a recursive processing step results
+        in the failure of this processing step.</p>
+    </section>
+
+    <section id="convert-document">
+      <h3>Converting a <a>YAML document</a></h3>
+
+      <p>From the <a data-cite="YAML#">YAML grammar</a>,
+        a YAML document MAY be proceeded by a
+        <a data-cite="YAML#rule-l-document-prefix">Document Prefix</a>
+        If a document contains a <a data-cite="YAML#rule-c-byte-order-mark">byte order mark</a>, it MUST
+        indicate the UTF-8 character encoding.
+        Otherwise a processing error has been found, and processing is aborted.</p>
```

Comment:
In general, we're going to need to be more explicit on error cases, along the lines of the JSON-LD API. This may be considered an extension to that API, and integrating with that has its own challenges.